cfme/control/explorer/actions.py

```diff
@@ -0,0 +1,228 @@
+        self.vcenter_atrr_value = kwargs.get("vcenter_atrr_value")
+        self.tag = kwargs.get("tag")
+
+    def __str__(self):
+        return self.description
+
+    def create(self):
+        "Create this Action in UI."
+        view = navigate_to(self, "Add")
+        view.fill({
+            "description": self.description,
+            "action_type": self.action_type,
+            "snapshot_name": self.snapshot_name,
+            "analysis_profile": self.analysis_profile,
+            "snapshot_age": self.snapshot_age,
+            "alerts_to_evaluate": self.alerts_to_evaluate,
+            "parent_type": self.parent_type,
+            "categories": self.categories,
+            "cpu_number": self.cpu_number,
+            "memory_amount": self.memory_amount,
+            "email_sender": self.email_sender,
+            "email_recipient": self.email_recipient,
+            "vcenter_atrr_name": self.vcenter_atrr_name,
+            "vcenter_atrr_value": self.vcenter_atrr_value,
+            "tag": self.tag
+        })
+        view.add_button.click()
+        view = self.create_view(ActionDetailsView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        view.flash.assert_message('Action "{}" was added'.format(self.description))
+
+    def update(self, updates):
+        """Update this Action in UI.
+
+        Args:
+            updates: Provided by update() context manager.
+            cancel: Whether to cancel the update (default False).
+        """
+        view = navigate_to(self, "Edit")
+        changed = view.fill(updates)
+        if changed:
+            view.save_button.click()
+        else:
+            view.cancel_button.click()
+        view = self.create_view(ActionDetailsView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        if changed:
+            view.flash.assert_message(
+                'Action "{}" was saved'.format(updates.get("description", self.description)))
+        else:
+            view.flash.assert_message(
+                'Edit of Action "{}" was cancelled by the user'.format(self.description))
+
+    def delete(self, cancel=False):
+        """Delete this Action in UI.
+
+        Args:
+            cancel: Whether to cancel the deletion (default False).
+        """
+        view = navigate_to(self, "Details")
+        view.configuration.item_select("Delete this Action", handle_alert=not cancel)
+        if cancel:
+            assert view.is_displayed
+            view.flash.assert_no_error()
+        else:
+            view = self.create_view(ActionsAllView)
+            assert view.is_displayed
+            view.flash.assert_no_error()
+            view.flash.assert_message('Action "{}": Delete successful'.format(self.description))
+
+
+@navigator.register(Action, "Add")
+class ActionNew(CFMENavigateStep):
+    VIEW = NewActionView
+    prerequisite = NavigateToAttribute("appliance.server", "ControlExplorer")
```

Comment:
you can also register ControlExplorer as "All" and use NavigateToSibling instead.